CMakeLists.txt

```diff
@@ -354,7 +354,10 @@ IF(BUILD_PYTHON_INTERFACE)
 ENDIF()
 
 PKG_CONFIG_APPEND_LIBS(${PROJECT_NAME})
-PKG_CONFIG_APPEND_BOOST_LIBS(${BOOST_REQUIRED_COMPONENTS})
+FOREACH(boostlib ${BOOST_REQUIRED_COMPONENTS})
+PKG_CONFIG_APPEND_LIBS("boost_${boostlib}")
+ENDFOREACH(boostlib ${BOOST_REQUIRED_COMPONENTS})
+# PKG_CONFIG_APPEND_BOOST_LIBS(${BOOST_REQUIRED_COMPONENTS})
```

Comment:
Should we change the jrl cmakemodule directly?

Reply:
> Why this change?

Because the aboslute path is only present, after the other change, because boost::filesystem is not header only, and so to still link on all components, without having -L/path it's the only solution i saw.

Reply:
for cmeel packaging, which also avoid full paths, those are simply removed for now.
A better solution would be to have a side project properly writing .pc files for boost, in a clean way. I started something in this direction 2y ago, but didn't went very far.